.github/CODEOWNERS

```diff
@@ -4,11 +4,11 @@
 *      @dbt-labs/product-docs
```

Comment:
@dbeatty10 @dataders - do you want to remain codeowners for the building packages and materialization pages?

Reply:
You can peel both of us out 👍